### PR TITLE
Changed refresh token documentation

### PR DESCRIPTION
**Pull request by christian-bouyssi**
CURL command format was incorrect

### CODE REVIEW

**Review by BitOne — APPROVED** [review]

File: content/rest-api/authentication.md [hunk 1]
```diff
@@ -219,22 +219,26 @@ For security reasons, access tokens have a certain lifespan. A client applicatio
...
 #### Example
 
 **Request**
 ```bash
-curl -X POST http://your-host/api/oauth/v1/token \
-    -H "Content-Type: application/json" \
-    -H "Authorization: Basic M2UyaXFpbHEyeWd3azBjY2dvZ2tjd2NvOG9vc2Nra2trNGdrb2MwazRzOHMwNDR3c3M6NDRlY3Rlbm11ZHVzOGc4OHc0d2t3czg0MDQ0Y2t3MGs0dzRrZzBzb2tvc3M4NG9rbzg=" \
-    -d "grant_type"=refresh_token \
-    -d "refresh_token"=MDk2ZmIwODBkYmE3YjNjZWQ4ZTk2NTk2N2JmNjkyZDQ4NzA3YzhiZDQzMjJjODI5MmQ4ZmYxZjlkZmU1ZDNkMQ 
+curl --location --request POST 'http://my-favorite-pim.com/api/oauth/v1/token' \
```
Comment (ahocquard) [inline content/rest-api/authentication.md:235]: nothing to change here

File: content/rest-api/authentication.md [hunk 2]
```diff
@@ -219,22 +219,26 @@ For security reasons, access tokens have a certain lifespan. A client applicatio
...
+--header 'Content-Type: application/json' \
```
Comment (ahocquard) [inline content/rest-api/authentication.md:236]: same, `-H` was fine

**Review by ahocquard — APPROVED** [review]

File: content/rest-api/authentication.md (hunk 1 above)
Comment (christian-bouyssi) [reply content/rest-api/authentication.md:235]: :ok_hand:

File: content/rest-api/authentication.md (hunk 2 above)
Comment (christian-bouyssi) [reply content/rest-api/authentication.md]: Changed to letter arguments

**Pull request merged**